src/application.cpp

```diff
@@ -42,7 +50,7 @@ int Application::run() {
     window.setFramerateLimit(30);
 
     // set up environment
-    BasicMapGenerator mapGenerator = BasicMapGenerator(envColours, soybeanOverlays, rows, columns, 100, 20);
+    BasicMapGenerator mapGenerator = BasicMapGenerator(envColours, soybeanOverlays, params.rows, params.columns, params.bees, params.soybean_p);
```

Comment:
Map generator takes in an int from 0-100 for its soybean probability, whilst the Parameter value is a float from 0-1

Reply:
Okay, *100 has been added to params.soybean_p in application